docs/Customizing Look and Functionality/Customizing the Template/index.md

```diff
@@ -183,3 +235,18 @@ setGlobalProperties.setOnRPCResponseListener(new OnRPCResponseListener() {
 sdlManager.sendRPC(setGlobalProperties);
 ```
 !@
+@![javascript]
+```js
+const setGlobalProperties = new SDL.rpc.messages.SetGlobalProperties();
+setGlobalProperties.setMenuTitle('customTitle');
+// The image must be uploaded before referencing the image name here
+setGlobalProperties.setMenuIcon(<#Image#>);
+sdlManager.addRpcListener(SDL.rpc.enums.FunctionID.SetGlobalProperties, function (response) {
+        if (response instanceof SDL.rpc.RpcResponse && response.getSuccess()){
+            // Success
+        }
+    }
+});
+sdlManager.sendRpc(setGlobalProperties);
```

Comment:
Use the preferred async/await syntax to get responses to RPC requests. Check other parts of this and other PRs to make sure that style is used.
ex:
`const response = await sdlManager.sendRpc(new SDL.rpc.messages.UnregisterAppInterface());`